src/docs/self-hosted/index.mdx

```diff
@@ -10,11 +10,11 @@ Our recommendation is to download the [latest release of the self-hosted reposi
 
 ### Installing Behind a Proxy
 
-In some enterprise setups there is no direct Internet connection, so you must use an HTTP proxy server. How do you install Sentry in this environment? Let us assume: 
+In some enterprise setups there is no direct Internet connection, so you must use an HTTP proxy server. How do you install Sentry in this environment? Let us assume:
 
 1. Your Sentry installation is running on Linux.
-1. `http://proxy:3128` is your proxy address.
-1. `127.0.0.0/8` is the only network that should be accessed without a proxy.
+2. `http://proxy:3128` is your proxy address.
+3. `127.0.0.0/8` is the only network that should be accessed without a proxy.
```

Comment:
Fwiw, Markdown does the right thing with these even if they're all the same number.